portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -73,11 +73,21 @@ function createCommentElement(comment) {
 	deleteEl.setAttribute("onclick", `deleteCommentFromStorageById(${comment.id})`);
 	deleteEl.setAttribute("src", "images/icons/trash.svg");
 
-	const authorDateDeleteContainer = document.createElement("div");
-	authorDateDeleteContainer.appendChild(authorEl);
-	authorDateDeleteContainer.appendChild(deleteEl);
-	authorDateDeleteContainer.appendChild(dateEl);
-	children.push(authorDateDeleteContainer);
+	const scoreEl = document.createElement("span");
+	const score = comment.score.toFixed(2);
+	scoreEl.innerText = `Sentiment: ${score}`;
+
+	// Set class based on score -> The higher, the more "positive"
+	if (score >= 0.33) scoreEl.setAttribute("class", "score positive")
+	else if (score >= -0.33) scoreEl.setAttribute("class", "score neutral")
+	else scoreEl.setAttribute("class", "score negative");
+
+	const authorDateDeleteScoreContainer = document.createElement("div");
```

Comment:
Nit: maybe rename this to something more generic, like "commentContainer" so you don't have to keep making the name more and more verbose as you add things :P

Reply:
Yeah that's a really good point haha, at this rate this variable is going to span multiple lines!